numpy/core/src/umath/ufunc_object.c

```diff
@@ -6287,6 +6290,383 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
 }
 
 
+typedef struct {
+    PyArrayMethod_StridedLoop *strided_loop;
+    PyArrayMethod_Context *context;
+    NpyAuxData *auxdata;
+    /* Should move to flags, but lets keep it bools for now: */
+    npy_bool requires_pyapi;
+    npy_bool no_floatingpoint_errors;
+    PyArrayMethod_Context _full_context;
+    PyArray_Descr *_descrs[];
+} ufunc_call_info;
```

Comment:
This is a local structure?

Reply:
Yes it is local and partially exposed via the capsule and documented in the function docstring.
I was considering putting the public part into `experimental_dtype_api.h`, but it did not seem very useful it the API is expected to still change a bit with versions (and unlike `experimental_dtype_api.h` I don't want to expect compiling against a specific NumPy version).